emr/application/controllers/Request.php

```diff
@@ -11,42 +11,133 @@ public function __construct() {
     }
 
     public function index(){
+      if ($this->ion_auth->is_admin()) {
+        $data["userRole"] = "ADMIN";
+        $data["options"] = ["Sync Data", "Create User", "Edit Users", "Change Password", "Logout"];
+        $data["href"] = ["data", "auth/create_user", "auth", "auth/change_password", "auth/logout"];
+        $data["font"] = ["database","user-plus", "edit", "refresh", "sign-out"];
 
-     if ($this->ion_auth->is_admin()) {
-       $data["userRole"] = "ADMIN";
-       $data["options"] = ["Sync Data", "Create User", "Edit Users", "Change Password", "Logout"];
-       $data["href"] = ["data", "auth/create_user", "auth", "auth/change_password", "auth/logout"];
-       $data["font"] = ["database","user-plus", "edit", "refresh", "sign-out"];
-
-       $data["sideMenu"] = ["Calendar", "Screeners", "Buildings", "Requests"];
-       $data["link"] = ["main/index", "screeners", "billing", "insights", "activity", "reminders"];
-       $data["icon"] = ["calendar","user", "building", "exclamation-triangle", "bookmark-o", "check-square-o"];    
-     } elseif ($this->ion_auth->in_group("hostpial admin")) {
-       $data["userRole"] = "HOSPITAL ADMIN";
-       $data["options"] = ["Logout"];
-       $data["href"] = ["auth/logout"];
-       $data["font"] = ["refresh", "sign-out"];
-
-       $data["sideMenu"] = ["Calendar", "Screeners", "Buildings", "Requests"];
-       $data["link"] = ["main/index", "screeners", "billing", "insights", "activity", "reminders"];
-       $data["icon"] = ["calendar","user", "building", "exclamation-triangle", "bookmark-o", "check-square-o"];   
-    } else {
-      $data["userRole"] = "SCREENER";
-      $data["options"] = ["Logout"];
-      $data["href"] = ["auth/logout"];
-      $data["font"] = ["sign-out"];
-
-      $data["sideMenu"] = ["Calendar", "Availability"];
-      $data["link"] = ["main/index", "screeners/add"];
-      $data["icon"] = ["calendar","user"];
+        $data["sideMenu"] = ["Calendar", "Screeners", "Buildings", "Requests"];
+        $data["link"] = ["main/index", "screeners", "billing", "insights", "activity", "reminders"];
```

Comment:
why do you have `insights` here as a link?

Reply:
I removed insights!